src/module.c

```diff
@@ -981,46 +982,72 @@ RedisModuleCommandProxy *moduleCreateCommandProxy(RedisModuleCtx *ctx, const cha
     return cp;
 }
 
+/* Get an opaque structure, representing a module command, by command name.
+ * This structure is used in some of the command-related APIs.
+ *
+ * NULL is returned in case of the following errors:
+ * * Command not found
+ * * The command is not a module command
+ * * The command doesn't belong to the calling module
+ */
+RedisModuleCommand *RM_GetCommand(RedisModuleCtx *ctx, const char *name) {
```

Comment:
This will not be rendered as a list, because there is no blank line before the bullet list.

(Sorry I didn't see this before. You sneaked in this new API in the huge PR.)

Reply:
@zuiderkwast can you please make a PR?